src/content/guidelines/motion/basics.mdx

```diff
@@ -194,23 +196,32 @@ Use `exit-easing` when removing elements from view, such as closing a modal or t
 </Tab>
 </WebsiteTabs>
 
-| Curves        | Productive                   | Expressive                    |
-| ------------- | ---------------------------- | ----------------------------- |
-| CSS           | cubic-bezier(0.2, 0, 1, 0.9) | cubic-bezier(0.4, 0.14, 1, 1) |
-| After Effects | Outgoing 20%, incoming 0%    | Outgoing 40%, incoming 0%     |
+| Style      | IBM Motion package         | CSS                           | After Effects             |
+| ---------- | -------------------------- | ----------------------------- | ------------------------- |
+| Productive | `motion(exit, productive)` | cubic-bezier(0.2, 0, 1, 0.9)  | Outgoing 20%, incoming 0% |
+| Expressive | `motion(exit, expressive)` | cubic-bezier(0.4, 0.14, 1, 1) | Outgoing 40%, incoming 0% |
 
 An exception to exits: if an element leaves the view but stays nearby, ready to reappear upon user action, use standard easing instead. A good example of this is a side panel. The panel would leave the view, but slows down as it exits, implying that it would come to rest just outside the view, and ready to be recalled.
```

Comment:
I would keep the exception paragraph with the main paragraph for exist easing. Seamed a little out of place and easy to overlook below the table.